zerver/tests/test_messages.py

```diff
@@ -982,7 +987,8 @@ def test_send_message_as_not_superuser_to_different_domain(self):
     def test_send_message_as_superuser_to_domain_that_dont_exist(self):
         # type: () -> None
         email = "emailgateway@zulip.com"
-        user = get_user_profile_by_email(email)
+        realm = get_realm("zulip")
+        user = get_user(email, realm)
```

Comment:
this should probably be `system_bot(settings.EMAIL_GATEWAY_BOT)`, since this one is a system bot.